docker/build/base.aarch64.dockerfile

```diff
@@ -1,14 +1,77 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
-LABEL version="1.2"
+ARG CUDA_LITE
+ARG CUDNN_VERSION
+ARG TENSORRT_VERSION
 
-ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends gnupg2 curl ca-certificates \
+    && echo "deb [trusted=yes] http://172.17.0.1:8080/arm64/ ./" > /etc/apt/sources.list.d/cuda.list \
```

Comment:
Does it work for others?

Reply:
Nope. This requires one to have local apt repository setup before running docker build. General users are not expected to build this image themselves. And Document on how to setup local apt repository should be provided in later PRs.